examples/chart/teleport-cluster/values.schema.json

```
@@ -199,7 +200,27 @@
                             "default": "Issuer"
                         }
                     }
-                }
+                },
+                "podDisruptionBudget": {
+                    "$id": "#/properties/highAvailability/properties/podDisruptionBudget",
+                    "type": "object",
+                    "required": [
+                        "enabled",
+                        "minAvailable"
+                    ],
+                    "properties": {
+                        "enabled": {
+                            "$id": "#/properties/highAvailability/properties/podDisruptionBudget/properties/enabled",
+                            "type": "boolean",
+                            "default": false
+                        },
+                        "minAvailable": {
+                            "$id": "#/properties/highAvailability/properties/podDisruptionBudget/properties/minAvailable",
+                            "type": "integer",
+                            "default": 1
+                        }
+                    }
+                },
```

```suggestion
                }
```

Tests are failing, I think this is probably why.

---

:facepalm: my bad!